### PR TITLE
feat: polymer devnet fast chain mode

### PR DESCRIPTION
**Pull request by qinlz2**
In this PR
* introduce polymer devnet mode
* handle race conditions around genesis block
* do not error out on known condition

### CODE REVIEW

**Comment by qinlz2** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #25** <a href="https://app.graphite.dev/github/pr/polymerdao/prysm/25" target="_blank">Graphite</a>  👈
    * **PR #26** <a href="https://app.graphite.dev/github/pr/polymerdao/prysm/26" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/polymerdao/prysm/25?utm_source=stack-comment).

File: beacon-chain/core/epoch/precompute/justification_finalization.go [hunk 1]
```diff
@@ -55,6 +57,11 @@ func UnrealizedCheckpoints(st state.BeaconState) (*ethpb.Checkpoint, *ethpb.Chec
 //	  current_target_balance = get_attesting_balance(state, current_attestations)
 //	  weigh_justification_and_finalization(state, total_active_balance, previous_target_balance, current_target_balance)
 func ProcessJustificationAndFinalizationPreCompute(state state.BeaconState, pBal *Balance) (state.BeaconState, error) {
+	if features.Get().PolymerDevnetMode {
```
Comment (7AC) [inline beacon-chain/core/epoch/precompute/justification_finalization.go:60]: can we call it something other than polymer? feels like it's a generic feature

**Review by 7AC — DISMISSED** [review]
One minor comment, also maybe we should keep this in this branch instead of merging since it's non standard and unrelated to light client support

**Comment by qinlz2** [comment]
Let's just keep it in the separate branch.  

The code doesn't target for the main stream `prysm`

**Pull request closed without merging**